dependencies/precise/hammer_cli_foreman_docker/install

```diff
@@ -0,0 +1,2 @@
+config/foreman_docker.yml     etc/hammer/cli.modules.d
+locale/en                     usr/share/locale
```

Comment:
@mbacovsky please ensure you create + sync all locales in the next version, as translations exist already